### PR TITLE
merge development branch

### PR DESCRIPTION
**Pull request by Nuranazm**

### CODE REVIEW

**Comment by alisher-zhuman** [comment]
pishem podrobnee commenty

File: package-lock.json
Comment (alisher-zhuman) [inline package-lock.json:1]: rename branch to feature/notification

File: src/App.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import './App.css'
+import React from 'react'
```
Comment (alisher-zhuman) [inline src/App.jsx:1]: delete react

File: src/components/Notification.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import 'react-toastify/dist/ReactToastify.css'
```
Comment (alisher-zhuman) [inline src/components/Notification.jsx:1]: style yldyirak

File: src/components/Notification.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import { ToastContainer } from 'react-toastify'
+import { styled } from '@mui/material'
+
+const Notification = () => (
+   <StyledToastContainer icon={false} position="top-left" autoClose={1000} />
+)
```
Comment (alisher-zhuman) [inline src/components/Notification.jsx:7]: otstup

File: src/components/Notification.jsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import 'react-toastify/dist/ReactToastify.css'
+import { ToastContainer } from 'react-toastify'
+import { styled } from '@mui/material'
+
+const Notification = () => (
+   <StyledToastContainer icon={false} position="top-left" autoClose={1000} />
```
Comment (alisher-zhuman) [inline src/components/Notification.jsx:6]: delete position
and autoClose

File: src/components/Notification.jsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+)
+export default Notification
+
+const StyledToastContainer = styled(ToastContainer)(() => ({
+   '& .Toastify__toast--error': {
+      backgroundColor: '#FFF9F8',
+   },
```
Comment (alisher-zhuman) [inline src/components/Notification.jsx:13]: otstupyyy

File: src/index.jsx [hunk 6]
```diff
@@ -3,12 +3,14 @@ import ReactDOM from 'react-dom/client'
...
 import Themes from './components/UI/Themes'
 import App from './App'
+import Notification from './components/Notification'
 
 const root = ReactDOM.createRoot(document.getElementById('root'))
 
 root.render(
    <StrictMode>
       <Themes>
+         <Notification />
```
Comment (alisher-zhuman) [inline src/index.jsx:13]: otstup

File: src/utils/helpers/index.js
Comment (alisher-zhuman) [inline src/utils/helpers/index.js:1]: notification.js

File: src/utils/helpers/index.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
+import { styled } from '@mui/material'
+import { toast } from 'react-toastify'
+
+export const showToast = ({ message, status }) => {
```
Comment (alisher-zhuman) [inline src/utils/helpers/index.js:4]: autoClose, position , duration kelet propston

File: src/utils/helpers/index.js [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+   toast[status](<StyledMessage>{message}</StyledMessage>, {
+      autoClose: 3000,
+      icon: false,
```
Comment (alisher-zhuman) [inline src/utils/helpers/index.js:7]: otstup

File: src/utils/helpers/index.js [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+      style: {
+         borderLeft: '10px',
+         borderLeftColor: status === 'error' ? 'red' : 'green',
+         borderLeftStyle: 'solid',
+         borderRadius: 0,
+      },
+   })
+}
+
+const StyledMessage = styled('p')(({ theme }) => ({
```
Comment (alisher-zhuman) [inline src/utils/helpers/index.js:17]: typograpgy

File: src/utils/helpers/index.js (hunk 7 above)
Comment (alisher-zhuman) [inline src/utils/helpers/index.js:4]: defolt status and message berip koiunuzchu

**Comment by alisher-zhuman** [comment]
box shadow zametnyi kylynyschy ato nezamento

File: package-lock.json
Comment (Nuranazm) [reply package-lock.json:1]: > feature/notification

ok

File: src/App.jsx (hunk 1 above)
Comment (Nuranazm) [reply src/App.jsx:1]: ok

File: src/components/Notification.jsx (hunk 2 above)
Comment (Nuranazm) [reply src/components/Notification.jsx:1]: ok

File: src/components/Notification.jsx (hunk 3 above)
Comment (Nuranazm) [reply src/components/Notification.jsx:7]: ok

File: src/components/Notification.jsx (hunk 4 above)
Comment (Nuranazm) [reply src/components/Notification.jsx:6]: ok

File: src/components/Notification.jsx (hunk 5 above)
Comment (Nuranazm) [reply src/components/Notification.jsx:13]: ok

File: src/index.jsx (hunk 6 above)
Comment (Nuranazm) [reply src/index.jsx:13]: здесь не получается сделать

File: src/utils/helpers/index.js
Comment (Nuranazm) [reply src/utils/helpers/index.js:1]: ok

File: src/utils/helpers/index.js (hunk 7 above)
Comment (Nuranazm) [reply src/utils/helpers/index.js:4]: ok

File: src/utils/helpers/index.js (hunk 8 above)
Comment (Nuranazm) [reply src/utils/helpers/index.js:7]: не получается

File: src/utils/helpers/index.js (hunk 9 above)
Comment (Nuranazm) [reply src/utils/helpers/index.js:17]: ок

File: src/utils/helpers/index.js (hunk 7 above)
Comment (Nuranazm) [reply src/utils/helpers/index.js:4]: ок

**Pull request merged**